app/retail/management/commands/sync_ideas.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import requests
+from retail.models import Idea
+
+logging.basicConfig(level=logging.DEBUG)
+
+logger = logging.getLogger(__name__)
+
+logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+class Command(BaseCommand):
```

Comment:
Two new lines between `logging.getLogger` and `class Command` - Also, `logger` doesn't appear to be used to pass logging anywhere.